pubsub-mapped-api/src/main/java/com/google/pubsub/clients/ProducerThread.java

```diff
@@ -0,0 +1,64 @@
+package com.google.pubsub.clients;
```

Comment:
All source files need to have the correct license. See the connector or loadtest framework for examples.